lineman/app/components/navbar/navbar_user_options.coffee

```diff
@@ -3,5 +3,9 @@ angular.module('loomioApp').directive 'navbarUserOptions', ->
   restrict: 'E'
   templateUrl: 'generated/components/navbar/navbar_user_options.html'
   replace: true
-  controller: ($scope, CurrentUser) ->
+  controller: ($scope, CurrentUser, Records, $http) ->
```

Comment:
No need for Records here, I don't think.

HMM the $http service.. is there a way we can move this call into the RestfulClient, so we don't have the '/api/v1' endpoint floating around in more than one place? We've done a really great job of keeping our api calls threaded through a single class so far.